_inc/client/plans/product-selector.jsx

```diff
@@ -46,6 +46,14 @@ class ProductSelector extends Component {
 			},
 		} );
 
+		const renewalDate = __( 'Renews on %(renewalDate)s', {
```

Comment:
We should also address this on the "My Plan" page:

![](https://cldup.com/PgG7vu9ngx.png)

That being said, it might make sense to create a utility function somewhere to reuse the logic in both places.

Reply:
Created a central component in 907553f35440635440ec58a9dc7ad4e9657801d5, and also added logic if the expiry date is in the past.